SpatialGDK/Source/SpatialGDK/Private/Utils/SpatialStatics.cpp

```diff
@@ -232,17 +232,15 @@ bool USpatialStatics::IsActorGroupOwnerForClass(const UObject* WorldContextObjec
 
 	if (const USpatialNetDriver* SpatialNetDriver = Cast<USpatialNetDriver>(World->GetNetDriver()))
 	{
-		// Calling IsActorGroupOwnerForClass before NotifyBeginPlay has been called (when NetDriver is ready) is invalid.
-		if (!SpatialNetDriver->IsReady())
-		{
-			UE_LOG(LogSpatial, Error,
-				   TEXT("Called IsActorGroupOwnerForClass before NotifyBeginPlay has been called is invalid. Actor class: %s"),
-				   *GetNameSafe(ActorClass));
-			return true;
-		}
-
 		if (const ULayeredLBStrategy* LBStrategy = GetLayeredLBStrategy(SpatialNetDriver))
 		{
+			if (!LBStrategy->IsReady())
```

Comment:
According to @m-samiec this is all we really care about. Fixing this by making `SpatialNetDriver` ready is really messy because of ordering issues, as this is called as part of the routine in which `SpatialNetDriver` is made ready (but is set to ready after this call).

Reply:
(Look at a previous commit for the netdriver ready approach if you are interested)